pkg/logcli/output/default_test.go

```diff
@@ -0,0 +1,88 @@
+package output
+
+import (
+	"testing"
+	"time"
+
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDefaultOutput_Format(t *testing.T) {
```

Comment:
You could test whether labels are padded correctly as well here ;)

Reply:
From my perspective, it's what the test `max labels length longer than input labels` does. Do you have something different in mind?

Reply:
I thought of pushing some lines and checking whether they are properly aligned ... nice to have, but the current one suffices as well

Reply:
I'm always in favour of having a good tests coverage. Let me work on it.

Reply:
What do you think about the approach [taken here](https://github.com/grafana/loki/pull/821/commits/50e4a1ae8909f96a4e7de913b7ffcb1423bcb898)?

Reply:
That's pretty neat! Thanks :)